packages/contract-addresses/CHANGELOG.json

```diff
@@ -4,10 +4,13 @@
         "changes": [
             {
                 "note": "Updated Ethereum FQT to enable RFQT V2",
-                "pr": 598
+                "pr": 604
+            },
+            {
```

Comment:
nit: The second item isn't necessary (not an interesting thing for the package consumer)

Reply:
pr isnt required?